tests/desktop_web/test_performance_issues.py

```diff
@@ -0,0 +1,19 @@
+import time
+import pytest
+import sentry_sdk
+
+def test_nplusone(desktop_web_driver, endpoints, random, batch_size, sleep_length):
```

Comment:
So I know you're just following existing convention but wondering if we should rename both to `nplusone_api_call` to disambiguate.

Reply:
yep, renaming 👍